docs: list more packages using `fdir`

benmccann opened the pull request:
There are a few well-known packages that use `fdir`. I listed them first because I've heard of them and haven't really heard of most of the other ones in the list

https://www.npmjs.com/package/miniprogram-to-uniapp has only 28 downloads / week, so I removed it

I left `astro-i18next`, which has about 1,800 downloads/week, but wasn't sure if we still needed it now that `astro` itself is being listed

thecodrr commented on README.md:
```diff
@@ -77,15 +77,18 @@ Please check the benchmark against the latest version [here](/BENCHMARKS.md).
-8. [yassinedoghri/astro-i18next](https://github.com/yassinedoghri/astro-i18next)
-9. [selfrefactor/rambda](https://github.com/selfrefactor/rambda)
+1. [withastro/astro](https://github.com/withastro/astro)
+2. [pulumi/pulumni](https://github.com/pulumi/pulumi)
```
Wrong spelling?

thecodrr commented on README.md:
```diff
@@ -77,15 +77,18 @@ Please check the benchmark against the latest version [here](/BENCHMARKS.md).
-7. [FredKSchott/snowpack](https://github.com/FredKSchott/snowpack)\*
-8. [yassinedoghri/astro-i18next](https://github.com/yassinedoghri/astro-i18next)
-9. [selfrefactor/rambda](https://github.com/selfrefactor/rambda)
+1. [withastro/astro](https://github.com/withastro/astro)
```
I searched their codebase and `fdir` doesn't seem to be used anymore: https://github.com/search?q=repo%3Awithastro%2Fastro+fdir&type=code

benmccann commented:
sorry I missed your comments earlier. I've addressed them all and added a new item `rollup/plugins`, which just merged a PR of mine to switch to `fdir`
